[2024-08-02T19:27:24.822+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: finetune_gpt_v1.transform_data manual__2024-08-02T19:27:13.963550+00:00 [queued]>
[2024-08-02T19:27:24.837+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: finetune_gpt_v1.transform_data manual__2024-08-02T19:27:13.963550+00:00 [queued]>
[2024-08-02T19:27:24.838+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 6
[2024-08-02T19:27:24.856+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): transform_data> on 2024-08-02 19:27:13.963550+00:00
[2024-08-02T19:27:24.865+0000] {standard_task_runner.py:60} INFO - Started process 162 to run task
[2024-08-02T19:27:24.871+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'finetune_gpt_v1', 'transform_data', 'manual__2024-08-02T19:27:13.963550+00:00', '--job-id', '105', '--raw', '--subdir', 'DAGS_FOLDER/finetune_gpt.py', '--cfg-path', '/tmp/tmps4qru4a9']
[2024-08-02T19:27:24.875+0000] {standard_task_runner.py:88} INFO - Job 105: Subtask transform_data
[2024-08-02T19:27:25.038+0000] {task_command.py:423} INFO - Running <TaskInstance: finetune_gpt_v1.transform_data manual__2024-08-02T19:27:13.963550+00:00 [running]> on host 763b10b8ea7f
[2024-08-02T19:27:27.335+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='finetune_gpt_v1' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-02T19:27:13.963550+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-02T19:27:13.963550+00:00'
[2024-08-02T19:27:28.759+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-08-02T19:27:28.945+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=finetune_gpt_v1, task_id=transform_data, execution_date=20240802T192713, start_date=20240802T192724, end_date=20240802T192728
[2024-08-02T19:27:29.213+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-08-02T19:27:29.309+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
